include/deal.II/grid/grid_generator.h

```diff
@@ -89,8 +89,8 @@ namespace GridGenerator
    *
    * @note The triangulation needs to be void upon calling this function.
    */
-  template <int dim>
-  void subdivided_hyper_cube (Triangulation<dim>  &tria,
+  template <int dim, int spacedim>
+  void subdivided_hyper_cube (Triangulation<dim,spacedim>  &tria,
                               const unsigned int   repetitions,
                               const double         left = 0.,
                               const double         right= 1.);
```

Comment:
Please document how `left` and `right` are interpreted if `dim<spacedim`. Are only the first `dim` components set to be between left and right? All components?